Base/usr/share/man/man1/tail.md

```diff
@@ -1,24 +1,39 @@
 ## Name
 
-tail
+tail - Print the end of a file
 
 ## Synopsis
 
-```sh
-$ tail [--follow] [--lines number] <file>
+```**sh
+$ tail [options]... [file]
 ```
 
 ## Description
 
-Print the end ('tail') of a file.
+`tail` prints the specified number (or 10) of lines at the end of `file` (or standard input).
 
-## Options:
+## Options
 
 * `-f`, `--follow`: Output data as it is written to the file
-* `-n number`, `--lines number`: Fetch the specified number of lines
+* `-n number`, `--lines number`: Print the specified number of lines
+
+## Arguments
+
+* `file`: Target file, or standard input
 
-## Arguments:
+## Examples
+
+Print the last 10 lines of README.md:
+```sh
+$ tail README.md
+```
 
-* `file`: File path
+Print the last 42 lines of todo.txt:
+```sh
+$ tail -n42 todo.txt
+```
 
-<!-- Auto-generated through ArgsParser -->
+Print the last lines as they are written to logs.log:
+```sh
+$ tail -f logs.log
+```
```

Comment:
A "See also" with `head` referenced wouldn't hurt, as those two commands are mirrors. Also, `cat` as the "noop" version of head/tail.